<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SensorPropertiesDialog</class>
 <widget class="QDialog" name="SensorPropertiesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sensor Properties</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,1,0" columnstretch="1,0,0,1">
   <item row="1" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBoxAlerts">
     <property name="title">
      <string>E&amp;nable temperature alerts for this sensor</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout2" stretch="0,0,1">
      <item>
       <widget class="QLabel" name="label2">
        <property name="text">
         <string>Thresho&amp;ld:</string>
        </property>
        <property name="buddy">
         <cstring>spinBoxThreshold</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxThreshold">
        <property name="suffix">
         <string> Â°C</string>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pushButtonOk">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="pushButtonCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <spacer name="horizontalSpacer4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout1" stretch="0,0,1">
     <item>
      <widget class="QLabel" name="label1">
       <property name="text">
        <string>Color:</string>
       </property>
       <property name="buddy">
        <cstring>comboBoxColors</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ColorComboBox" name="comboBoxColors"/>
     </item>
     <item>
      <spacer name="horizontalSpacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorComboBox</class>
   <extends>QComboBox</extends>
   <header>ColorComboBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboBoxColors</tabstop>
  <tabstop>groupBoxAlerts</tabstop>
  <tabstop>spinBoxThreshold</tabstop>
  <tabstop>pushButtonOk</tabstop>
  <tabstop>pushButtonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>SensorPropertiesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
